src/components/CSVForm.tsx

```diff
@@ -44,7 +44,7 @@ export const CSVForm = (props: CSVFormProps): JSX.Element => {
             marginRight: 3,
           }}
         />{" "}
-        {tokenList.get(payment.tokenAddress)?.symbol || payment.tokenAddress}
```

Comment:
Very cool! What would happen here now if the symbol was unavailable? Do we will have the possibility or need to display the token address in such a case? Or has this been completely avoided?

Reply:
It was fully avoided but now I unavoided it so its back to the old stuff.